apps/settings/build/build.js

```diff
@@ -52,7 +52,8 @@ SettingsAppBuilder.prototype.writeDeviceFeaturesJSON = function(options) {
                            'device-features.json');
   var defaultContent = {
     ambientLight: true,
-    vibration: true
+    vibration: true,
+    usbHotProtocolSwitch: false
```

Comment:
property to control if allowing to change transfer protocol when the usb transfer switch is enabled.